scripts/qa_crowbarsetup.sh

```diff
@@ -2336,6 +2336,7 @@ function oncontroller_testsetup()
         pushd /var/lib/openstack-tempest-test
         echo 1 > /proc/sys/kernel/sysrq
         ./run_tempest.sh -N $tempestoptions 2>&1 | tee tempest.log
+        testr last --subunit | subunit-1to2 | tee tempest.subunit.log
```

Comment:
you need to first save tempestret before executing anything else.. swap those two lines..